x-pack/plugins/aiops/server/routes/explain_log_rate_spikes.ts

```diff
@@ -591,87 +631,98 @@ export const defineExplainLogRateSpikesRoute = (
 
         loaded += PROGRESS_STEP_HISTOGRAMS_GROUPS;
 
-        logDebugMessage('Fetch field/value histograms.');
+        logDebugMessage(`Fetch ${changePoints.length} field/value histograms.`);
 
         // time series filtered by fields
         if (changePoints && overallTimeSeries !== undefined) {
```

Comment:
If changePoints is always defined, should this if condition checks for changePoints.length > 0 instead?

Reply:
Good catch! Must be a leftover from a previous refactor where `changePoints` was originially defined with `let`. Fixed in [9f81a3e](https://github.com/elastic/kibana/pull/143683/commits/9f81a3e6a882a29c177cf316300c43a18cd410a6).